lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -150,75 +144,42 @@ abstract class SqlParserTestBase : TestBase() {
-        // Check round trip for actual: SexpElement -> astStatement -> SexpElement
-        assertRoundTripIonElementToPartiQlAst(actualElement, expectedSexpAst)
-
+    private fun pigDomainAssert(actualStatement: PartiqlAst.Statement, expectedElement: SexpElement) {
         // Check equal for actual and expected in SexpElement format
-        // Check round trip for actual: astStatement -> SexpElement -> astStatement
-        // Check equals for actual value after round trip transformation: ExprNode -> astStatement -> SexpElement -> astStatement -> ExprNode
-        assertRoundTripPartiQlAstToExprNode(actualStatement, expectedSexpAst, actualExprNode)
-    }
-
-    private fun assertRoundTripPartiQlAstToExprNode(actualStatement: PartiqlAst.Statement, expectedSexpAst: IonElement, actualExprNode: ExprNode) {
-        // Run additional checks on the resulting PartiqlAst instance
-
-        // None of our test cases are wrapped in (query <expr>), so extract <expr> from that out
         val actualElement = unwrapQuery(actualStatement)
-        assertEquals(expectedSexpAst, actualElement)
+        assertEquals(expectedElement, actualElement)
 
-        // Convert the the IonElement back to the PartiqlAst instance and assert equivalence
-        val transformedActualStatement = PartiqlAst.transform(actualStatement.toIonElement()) as PartiqlAst.Statement
-        assertEquals(actualStatement, transformedActualStatement)
+        // Check equal after transformation: PIG AST -> SexpElement -> PIG AST
+        assertRoundTripPigAstToSexpElement(actualStatement)
 
-        // Convert from the PIG instance back to ExprNode and assert the result is the same as actualExprNode.
-        val transformedActualExprNode = MetaStrippingRewriter.stripMetas(transformedActualStatement.toExprNode(ion))
-        assertEquals(MetaStrippingRewriter.stripMetas(actualExprNode), transformedActualExprNode)
-    }
-
-    private fun assertRoundTripIonElementToPartiQlAst(actualElement: SexpElement, expectedElement: SexpElement) {
-        // #1 We can transform the actual PartiqlAst element.
+        // Check equal for actual and expected in transformed astStatement: astStatment -> SexpElement -> astStatement
         val transformedActualStatement = PartiqlAst.transform(actualElement)
-
-        // #2 We can transform the expected PartiqlAst element.
         val transformedExpectedStatement = PartiqlAst.transform(expectedElement)
-
-        // #3 The results of both transformations match.
         assertEquals(transformedExpectedStatement, transformedActualStatement)
 
-        // #4 Re-transforming the actual PartiqlAst element and check if it matches the expected AST.
+        // Check round trip for actual: SexpElement -> astStatement -> SexpElement
         val reserializedActualElement = transformedActualStatement.toIonElement()
         assertEquals(expectedElement, reserializedActualElement)
```

Comment:
This checks equal for actual value after round trip transformation: `ExprNode` -> PIG AST -> `SexpElement` -> PIG AST -> `ExprNode`. Since it is too complicated and we are deprecating `ExprNode`, I think we can remove this one.

Reply:
If I'm understanding correctly, previously, the check was:

```
ExprNode -> PIG AST -> SexpElement -> PIG AST -> ExprNode
```

In this PR, you changed it to:

```
PIG AST -> SexpElement -> PIG AST
```

Basically removing the `ExprNode` conversions at the start and end? If so, then I think it should be fine since the conversion to and from `ExprNode` is checked in `assertRoundTripPigAstToExprNode`

Reply:
Basically, that's what I mean here. Since we are deprecating `ExprNode`, this check seems too complicated to perform.